rotkehlchen/premium/sync.py

```diff
@@ -169,7 +156,19 @@ def maybe_upload_data_to_server(self, force_upload: bool = False) -> bool:
                 )
                 return False
 
-            data_bytes_size = len(base64.b64decode(b64_encoded_data))
+            data, our_hash = self.data.compress_and_encrypt_db()
+
+            log.debug(
```

Comment:
The reason you moved it afterwards is that the length now is in bytes and as such does not need the b64 encryption?

Reply:
yes and also there are other things checked before the size so moved it to the place where is required